docs/developers/add-new-bidder.md

```diff
@@ -33,6 +33,14 @@ Bidder implementations are scattered throughout several files.
 
 Bidder implementations may assume that any params have already been validated against the defined json-schema.
 
+###Long form video support
```

Comment:
I believe you need a space here after `###` for it to be correctly formatted. Without the space it doesn't show `Long form video support` as a heading but just normal text with `###` prepended to it.

Reply:
Updated!